scripts/qa_crowbarsetup.sh

```diff
@@ -450,11 +450,11 @@ function iscloudver()
         local milestone=${v#*M}
         v=${v%M*}
         if [[ "$ver" -eq "$v" ]] && [[ $cloudsource =~ ^M[0-9]+$ ]] ; then
-            [ "${cloudsource#*M}" $operator "$milestone" ]
+            test "${cloudsource#*M}" $operator "$milestone"
```

Comment:
Following @aspiers's suggestion I ran the script through [shellcheck](https://www.shellcheck.net/) which can't handle the test expression using the `[` form, and terminates with a nonsensical error message after this line. This is a bug in `shellcheck` to be sure. Both forms are equivalent.